puppet/modules/slave/manifests/packaging/debian.pp

```diff
@@ -6,6 +6,8 @@
   Stdlib::HTTPUrl $debian_mirror = 'http://deb.debian.org/debian/',
   Stdlib::HTTPUrl $ubuntu_mirror = 'http://archive.ubuntu.com/ubuntu/',
 ) {
+  include rsync
```

Comment:
Should this be part of `freight::uploader` or is it also used before uploading?

Reply:
nah, that's the place!